src/main/java/duke/Items.java

```diff
@@ -0,0 +1,135 @@
+package duke;
+
+import duke.task.Task;
+import java.util.ArrayList;
+
+/**
+ * The items in the bot
+ * responsible for adding things to the list of items
```

Comment:
Perhaps need a full stop at the end? 🤔
And I noticed the same issue in several other places below.